web/docs/Backend Starters/Decoupled Drupal/configuring-existing-project.md

```diff
@@ -0,0 +1,191 @@
+---
+id: 'configuring-existing-project'
+title: 'Configuring an Existing Drupal Project'
+slug: '/backend-starters/decoupled-drupal/configuring-an-existing-project'
+sidebar_position: 3
+---
+
+While we offer a
+[back-end starter project](backend-starters/decoupled-drupal/creating-a-new-project)
+to simplify the process of configuring a Drupal site for use with our front-end
+starter kits, you may instead prefer to use an existing Drupal project. Follow
+the steps below to configure an existing Drupal project to work with one of our
+front-end starter kits.
+
+## Before You Begin
+
+These instructions assume that you have already installed Drupal using your
+preferred method.
+
+The amount of necessary configuration will vary depending on the features you
+intend to use within your front-end starter kit. As a result, the instructions
+below are broken down into three related sections.
+
+## 1. Configuring Basic Builds
+
+If you only need to source content anonymously from your Drupal site, follow the
+steps below.
+
+### Add and Enable Dependencies
+
+- Run the following Composer commands:
+
+```bash
+composer config minimum-stability alpha
+composer require drupal/jsonapi_menu_items drupal/jsonapi_hypermedia drupal/decoupled_router
+```
+
+- Enable the media_library, jsonapi, jsonapi_menu_items_hypermedia, and
+  decoupled_router modules, which will also enable a number of dependencies.
+- Clear the Drupal cache.
+
+### Configure Images
+
+Our starter kits assume that you are using Drupal's core Media module to manage
+images. If you are instead using the default image field, you will need to make
+one of two possible adjustments:
+
+- Add a new field of type 'Media' to the Article content type. The resulting
+  field should have the label 'Media Image' which will result in the machine
+  name `field_media_image`.
+- Update your starter kit to use the default image field instead of the Media
+  field. Consult the
+  ['adapting for use with existing Drupal sites'](../../frontend-starters/nextjs/nextjs-drupal/troubleshooting#adapting-for-use-with-existing-drupal-sites)
+  entry within the Next.js + Drupal documentation for further instructions.
+
+### Ensure That Your Content Uses URL Aliases
```

Comment:
Does the URL Aliases are not suppose to be in a particular format like: `articles/node-title' ?

Reply:
The starter kits can actually handle path aliases in any format, as long as they have an alias.